src/screens/application/index.js

```diff
@@ -30,6 +30,10 @@ const Application = () => {
         isThemeReady,
         isPinCodeRequire,
       }) => {
+        if (showAnimation || !isReady || !isRenderRequire || !isThemeReady) {
```

Comment:
Mustafa moved this inside applicationscreen so that notification opening works, @mistikk, @ue

Reply:
otherwise darktheme goingto crush @feruzm